rialto_airflow/publish/openaccess.py

```diff
@@ -100,7 +104,8 @@ def write_contributions(snapshot) -> Path:
         "federally_funded",
     ]
 
```

Comment:
We are now writing the open-access files to a sub-folder of the snapshot directory so they don't get confused with files for other dashboards.